requirements.txt

```
@@ -14,7 +14,7 @@ google-cloud>=0.32.0
 google-cloud-bigquery<=3.0.0
 google-cloud-storage>=1.6.0
 gunicorn>=19.9.0
-joblib==0.15.1
+joblib==1.1.0
 MarkupSafe==2.0.1 # TODO: remove when #345 is fixed
```

```suggestion
MarkupSafe==2.0.1 # TODO: remove when #345 is fixed
```
joblib is not used by actinia-core, thus no need to install it.

joblib is used by `r.learn.ml2`, thus joblib (together with other dependencies such as scikit-learn) should be installed where `r.learn.ml2` is installed.